pkg/util/util.go

```diff
@@ -276,6 +281,11 @@ func kubeCreateSkipOrFailExisting(obj client.Object, skipOrFail bool) bool {
 		log.Printf("❌ Invalid: %s %q: %s\n", gvk.Kind, objKey.Name, err)
 		return false
 	}
+	statusErr, ok := err.(*errors.StatusError)
```

Comment:
this is a general function and you print always a very specific Admission Deny prefix to the error, is that always the case?